css/src/components/layout.scss

```diff
@@ -0,0 +1,110 @@
+$side-bar-width-wide: 4fr !default;
+
+.layout {
+	display: flex;
+	flex-direction: column;
+	min-height: 100vh;
+	overflow-x: hidden;
```

Comment:
So here's a bit of a fun question: to what extent should this layout use logical properties?

I was going to propose [`overflow-inline`](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-inline) here, and [`max-inline-size`](https://developer.mozilla.org/en-US/docs/Web/CSS/max-inline-size), but I'm unsure how much we actually expect this layout to adjust to vertical writing modes in practice.

Thoughts?

Reply:
Cool ideas! I also don't know. But I don't see harm it applying them here for these particular cases.

Reply:
They are applied (although `min-inline-size` not max))

Reply:
Whoops, typo on my part!